app/components/common/Navigation.vue

```diff
@@ -568,6 +575,42 @@ export default Vue.extend({
 #main-nav.navbar {
   background: white;
 
+  &.floating-nav {
+    position: absolute;
+    top: 0;
+    left: 50%;
+    margin: 0 auto;
+    transform: translate(-50%, -100%);
+    z-index: 10000;
+    border-radius: 50px;
+    padding: 10px;
+    box-shadow: 0 4px 20px rgba(0, 0, 0, 0.1);
+    transition: transform 1s ease;
+
+    &:has(.floating-nav-trigger:hover), &:hover {
+      transform: translate(-50%, 0);
+    }
+
+    &:hover > .floating-nav-trigger {
+      opacity: 0;
+      pointer-events: none;
+    }
+
+    .floating-nav-trigger {
+      width: 40px;
+      height: 5px;
+      z-index: 9000;
+      background-color: white;
+      border-radius: 2.5px;
+      position: absolute;
+      bottom: -15px;
```

Comment:
Probably need to change it to -5px as on playLevel, it shows on a existing element

<img width="272" height="86" alt="Screenshot 2025-08-27 at 3 01 20 PM" src="https://github.com/user-attachments/assets/1b0a2e70-8e98-4c5d-a3c5-f4d6a472e4de" />

Reply:
-5px looks good to me. changed